.gitignore

```diff
@@ -30,3 +30,7 @@
 *.exe
 *.out
 *.app
+
+#Data
+*.root
+*.art
```

Comment:
Please update your work patterns so that you never put these files in this directory; they belong in your work space on /mu2e/data or in dCache.  Please also remove the .art and .root lines from this file.  I want those files to be in your face because they don't belong there.  If you see them, immediately move them to your space on /mu2e/data.

Also please remove #Data .

Reply:
YEs, its more of a "just in case" these days, I've been burnt before with an accidental several GB .art file getting stuck in the .git history and it was a mess. I tend to do what you say these days